about.html

```diff
@@ -0,0 +1,142 @@
+      </ul>
+    </div>
+  </section>
+  <section id="nav-section" class="nav-section">
+    <div class="nav-social-container bc-dark d-flex d-justify-center d-none">
+      <ul class="nav-social desk-w d-flex d-justify-end">
+        <li class="nav-social-item mr-1"><i class="fab fa-facebook-f"></i></li>
+        <li class="nav-social-item mr-1"><i class="fab fa-twitter"></i></li>
+        <li class="nav-social-item mr-1">English</li>
+        <li class="nav-social-item mr-1">My Page</li>
+        <li class="nav-social-item">Logout</li>
+      </ul>
+    </div>
+
+    <nav id="navbar" class="nav-bar">
+      <div id="bars-menu" class="bars-menu d-none-desk">
+        <i class="fas fa-bars fa-2x"></i>
+      </div>
+      <div class="nav-list-container bc-white d-flex d-justify-between d-align-center d-none desk-w">
+        <a href="./index.html"><img class="nav-logo" src="./img/logos/ant_logo.png" alt=""></a>
+        <ul class="nav-list d-justify-between d-align-center d-none">
+          <li><a class="nav-links default-family fc-orange fw-700" href="./about.html">About</a></li>
+          <li><a class="nav-links default-family fc-darkgrey fw-700" href="#">Program</a></li>
+          <li><a class="nav-links default-family fc-darkgrey fw-700" href="#">Join</a></li>
+          <li><a class="nav-links default-family fc-darkgrey fw-700" href="#">Sponsor</a></li>
+          <li><a class="nav-links default-family fc-darkgrey fw-700" href="#">News</a></li>
+          <li><a class="nav-links nav-list-item-btn default-family fc-orange fw-700" href="#">MicroAnt Campain</a></li>
+        </ul>
+      </div>
+    </nav>
+  </section>
+
+  <section id="headline-section" class="headline-section d-flex d-justify-center pl-1">
+    <div class="headline-container d-flex d-column">
+      <h1 class="headline-desk-size headline-title default-family fw-900 fc-orange fs-48 mb-2">
+        MICROANT FILM FESTIVAL
+      </h1>
+      <p class="headline-desk-size headline-pbox default-family fw-500 fs-14 fc-greyish mb-1">
+        Ocelot egyptian mau siberian leopard or abyssinian . Leopard leopard so persian. Bobcat tom for siamese tom abyssinian for donskoy or leopard. Birman kitten panther for tom but savannah american shorthair. Kitten. Abyssinian mouser bengal american shorthair malkin. Manx savannah and siamese russian blue, sphynx. Turkish angora tiger yet burmese. Savannah savannah. Himalayan turkish angora bombay.
+      </p>
+      <h2
+        class="
+          headline-desk-size
+          headline-support
+          default-family
+          fw-900
+          fc-dark-greyish
+          fs-24
+          mb-1
+        "
+      >
+        2021.8.8(SAT) ~ 9(SUN)
+      </h2>
+      <h2
+        class="headline-desk-size headline-support default-family fw-500 fc-greyish fs-14 mb-1"
+      >
+        @ Zoom Conference Meeting, EveryWhere in the Globe.
+      </h2>
```

Comment:
- [ ] At the end of this section on the about page, there should be another text that refers to an email or other social connection method as described in the project requirements and the design template. Please let's add that 🙏 

_Take a look at the original design and compare it to yours._
![Screenshot](https://user-images.githubusercontent.com/47625407/128554140-33f95e35-eaa0-41c1-9af8-a08469db7fe5.png)